torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -226,9 +226,15 @@ def _init_device_mesh(
         return None
     if get_backend() == "fake" or not root_state.compute_device:
         return None
+    from torch.distributed.distributed_c10d import _get_group_tag
```

Comment:
Is this intentionally imported here and not at the top?

Reply:
not really, I planned to address the lazy init in a follow up PR so this import might not needed later, I can move it to top first.